components/prism-jsx.js

```diff
@@ -3,10 +3,20 @@
 var javascript = Prism.util.clone(Prism.languages.javascript);
 
 Prism.languages.jsx = Prism.languages.extend('markup', javascript);
-Prism.languages.jsx.tag.pattern= /<\/?[\w\.:-]+\s*(?:\s+[\w\.:-]+(?:=(?:("|')(\\?[\w\W])*?\1|[^\s'">=]+|(\{[\w\W]*?\})))?\s*)*\/?>/i;
+Prism.languages.jsx.tag.pattern= /<\/?[\w\.:-]+\s*(?:\s+[\w\.:-]*(?:=?(?:("|')(\\?[\w\W])*?\1|[^\s'">=]+|(\{[\w\W]*?\})))?\s*)*\/?>/i;
```

Comment:
The consequence is that you are allowing a whole bunch of weird stuff doing this. With the new regexp, `<div "">` would match, for example.

I'd prefer a more explicit version, that allows the spread operator. Something like...

```
/<\/?[\w\.:-]+\s*(?:\s+(?:[\w\.:-]+(?:=(?:("|')(\\?[\w\W])*?\1|[^\s'">=]+|(\{[\w\W]*?\})))?|\{\.{3}\w+\})\s*)*\/?>/i
```

or if you want to combine the spread part with the existing part using curly braces, then something like:

```
/<\/?[\w\.:-]+\s*(?:\s+(?:[\w\.:-]+(?:=(?:("|')(\\?[\w\W])*?\1|[^\s'">=]+))?|(?:[\w\.:-]+=)?\{[\w\W]*?\})\s*)*\/?>/i
```
though I think the first one is a little bit more readable, and might backtrack a bit less.

What do you think?